userspace/falco/falco.cpp

```diff
@@ -1438,7 +1061,7 @@ int falco_init(int argc, char **argv)
 			catch(sinsp_exception &e)
 			{
 				// If syscall input source is enabled and not through userspace instrumentation
-				if (!disable_syscall && !userspace)
+				if (enabled_sources.find(syscall_source) == enabled_sources.end() && !app.options().userspace)
```

Comment:
I may be wrong at this, but does not finding a source in the `enabled_sources` mean that the source is disabled? Shouldn't this be `!=` instead of `==`?

Reply:
You're right, this is backwards. I got the logic right earlier in line 1043 but not here. Fixed.